### ISSUE TITLE
Add subcommand to print rudimentary coverage statistics

### ISSUE THREAD

**Issue by kevinburkeomg** [codeclimate/test-reporter#343]
I'm trying to test whether results are being clobbered, and/or whether merge is working properly.

It would be helpful if there was a subcommand I could run that could print out rudimentary statistics about the combined code coverage file.

Alternatively, it would be good if the Code Climate website made it easy to see rudimentary information about code coverage on a branch, or on a single commit.

**Comment by wilson** [comment]
Thanks for the suggestion; I'm adding this to our product tracker.

Do you have an opinion on which details make the most sense to display?

Just to make sure I understand the use-case correctly, you'd like a subcommand to run after `sum-coverage` but before `upload-coverage`, to ensure that the proper payload is about to be sent?

**Comment by kevinburkeomg** [comment]
Yeah, basically. How many lines total, how many are covered, would probably suffice for now.
